app/src/main/java/com/sdp13epfl2021/projmag/notification/Constants.kt

```diff
@@ -0,0 +1,5 @@
+package com.sdp13epfl2021.projmag.notification
+
+const val BASE_URL = "https://fcm.googleapis.com"
+const val SERVER_KEY= "AAAAwNSBlt8:APA91bH7r-mQhoX9-KzML9N_86Hq1N6nZFO-cONdw7ghuG8SJHAmV4A3rZ-_TR8PEhxxGTGftPzThNVpYVT9_GF_a7PLobSobUdJ0pTSnUh5VvUl_p53Co-golonO3RhaXYzhUvJgSs2"
+const val  CONTENT_TYPE = "application/json"
```

Comment:
key is still here

Reply:
@giogio21 if you ignored the file you had to delete it before (or just `SERVER_KEY= "" `)

Reply:
Normaly it is good now